scripts/ex_fl_segway_ctrl.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+import numpy as np
+
+sys.path.append("..")
+
+from dyn_sim.ctrl.segway.fdbk_lin_segway_ctrl import FLPosSegwayController  # noqa: E402
+from dyn_sim.sim.simulator import SimulationEnvironment  # noqa: E402
+from dyn_sim.sys.planar.segway import Segway  # noqa: E402
+
+# SEGWAY #
+seg = Segway()
+
+# FDBK LIN CONTROLLER #
+flc = FLPosSegwayController(seg, 0.0, 1.0, 1.0)
+
+# SIMULATION ENVIRONMENT #
+simulator = SimulationEnvironment(seg, flc)
+
+if __name__ == "__main__":
+    x0 = np.array([1, 0, 0, 0])
+    sim_length = 15.0  # a long horizon here produces a pretty funny animation
+    n_frames = int(50 * sim_length + 1)
+    horizon = np.linspace(0, sim_length, n_frames)
+    t_sol, x_sol = simulator.simulate(x0, horizon)
+
+    # animate
+    # TODO implement 2d animation
```

Comment:
try this in the `simulator.py` file and lmk if it's compatible. github's current setup makes it so that i can't suggest this change on a non-edited file.

```
self._fig = plt.figure()
if self._sys._is3d:
  self._ax = Axes3D(self._fig)
  self._ax.set_proj_type("ortho")
  self._ax.grid(False)
  self._ax.set_xticks([])
  self._ax.set_yticks([])
  self._ax.set_zticks([])
  self._ax.set_xlim3d(xlim)
  self._ax.set_ylim3d(ylim)
  self._ax.set_zlim3d(zlim)
else:
  self._ax = Axes(self._fig)
  self._ax.grid(False)
  self._ax.set_xticks([])
  self._ax.set_yticks([])
  self._ax.set_xlim3d(xlim)
  self._ax.set_ylim3d(ylim)
```